charts/finrem-ccd-definitions/values.preview.template.yaml

```diff
@@ -57,7 +57,7 @@ finrem-cos:
   enabled: true
   java:
     imagePullPolicy: Always
-    image: hmctspublic.azurecr.io/finrem/cos:latest
+    image: hmctspublic.azurecr.io/finrem/cos:pr-1529
```

Comment:
Why is this being changed?

Reply:
@jthmcts This is to point to the COS PR and set up the preview environment (I would then remove it once it goes through testing)